README.md

```diff
@@ -11,31 +11,49 @@ Apply the plugin to your project following
 [`https://plugins.gradle.org/plugin/com.palantir.python.miniconda`](https://plugins.gradle.org/plugin/com.palantir.python.miniconda),
 and configure the associated extension:
 
+Minimal configuration:
 ```gradle
 miniconda {
-    bootstrapDirectoryPrefix = new File(System.getProperty('user.home'), '.miniconda')
-    buildEnvironmentDirectory = new File(buildDir, 'python')
-    minicondaVersion = '3.10.1'
-    packages = ['ipython-notebook']
+    minicondaVersion = '3.19.0'
+    packages = ['python']
 }
 ```
-
 Then invoke the `setupPython` task and use the resulting installation directory from `Exec` tasks:
 
 ```gradle
 task launchNotebook(type: Exec) {
     dependsOn 'setupPython'
-    executable "${miniconda.buildEnvironmentDirectory}/bin/ipython"
-    args 'notebook'
+    executable "${miniconda.buildEnvironmentDirectory}/bin/python"
+    args '-c', 'print("Hello world!)'
 }
 ```
 
 Options
 -------
 
-You can customize where the Miniconda installer script is downloaded from by adding a dependency to the
-`minicondaInstaller` configuration. By default, it's downloaded from
-[`http://repo.continuum.io`](http://repo.continuum.io).
+| Name | Default Value | Description | Optional |
+| ---- | ------------- | ----------- | -------- |
+| minicondaVersion | N/A | The miniconda version which you want to use. See [the miniconda repo](https://repo.continuum.io/miniconda/) | false
+| packages | N/A | The conda packages you want installed into your conda environment. This list must contain at least one argument. | false
+| bootstrapDirectoryPrefix | `new File(System.getProperty('user.home'), '.miniconda')` | The root directory to put the root install of miniconda. This helps performance by caching the root environment by `pythonVersion` and `minicondaVersion`. | true
+| buildEnvironmentDirectory | new File(buildDir, 'miniconda') | The directory to place your specific miniconda environment. | true
```

Comment:
Line 38 uses a code font for the example, this one uses a normal font. We should be consistent.